src/components/pages/Registration/InstructorRegistration/InstructorFlow_Step2.js

```diff
@@ -68,10 +57,10 @@ const InstrRegForm = () => {
 
   let history = useHistory();
 
+  //TO-DO: Implement axiosWithAuth once we've adjusted it to work with Auth0
   const onSubmit = evt => {
     evt.preventDefault();
-
-    axiosWithAuth(authState.idToken)
+    axios
```

Comment:
Doesn't this need to be `axios()`?

Reply:
No, I don't believe so. A typical axios call looks like axios.post('/path').then().catch()

Reply:
Ah, you're right. I got so used to seeing it as `axiosWithAuth` that I thought the parentheses were always there 😄